packages/node-app/src/controllers/giveaway.controller.ts

```diff
@@ -305,6 +348,20 @@ export const updateParticipant = async (req: Request, res: Response) => {
     }
     await giveaway.save();
 
+    const { id, title, description, prize, rules, image } = giveaway;
+    await GiveawayLog.create({
+      action: 'update-giveaway-participant',
+      author: req.user.email,
+      giveaway_id: id,
+      changes: {
+        title,
+        description,
+        prize,
+        rules,
+        image,
+      },
```

Comment:
These changes do not reflect what was changed exactly, here we should be able to save the new participant state or the notified flag